cob_helper_tools/scripts/auto_recover.py

```diff
@@ -29,6 +29,8 @@ class AutoRecover():
   def __init__(self):
     now = rospy.Time.now()
     self.em_state = 0
+    self.recover_emergency = rospy.get_parm('~recover_emergency', True)
+    self.recover_diagnostics = rospy.get_parm('~recover_diagnostics', False)
```

Comment:
does it make sense to use `True` as default for both

Reply:
we would need to adjust all the `components.yaml`...for testing this was easier....and it is the hardware-gentle and failure-agnostic way to do